appeals/web/src/server/lib/dates.js

```diff
@@ -22,6 +22,19 @@ export const dateIsValid = (year, month, day) => {
 	);
 };
 
+/**
+ * @param {number} year
+ * @param {number} month
+ * @param {number} day
+ * @returns {boolean}
+ */
+export const dateIsInTheFuture = (year, month, day) => {
+	const date = new Date(year, month - 1, day);
+	const now = new Date();
+
+	return dateIsValid(year, month, day) && now < date;
```

Comment:
It would be better to separate the `dateIsValid` check from this function for separation of concerns, you'd just need to call both functions from the custom validator in `date-input.validator.js`